src/test/java/uk/ac/ebi/atlas/experimentpage/markergenes/MarkerGenesDaoIT.java

```diff
@@ -86,9 +87,9 @@ void setUp() {
 
     @ParameterizedTest
     @MethodSource("ksForExperimentWithMarkerGenes")
-    void testExperimentsWithMarkerGenesAboveThreshold(int k) {
+    void testExperimentsWithMarkerGenesAboveThreshold(String k) {
         var markerGenesWithAveragesPerCluster =
-                subject.getMarkerGenesWithAveragesPerCluster(EXPERIMENT_ACCESSION_WITH_MARKER_GENES, k);
+                subject.getMarkerGenesWithAveragesPerCluster(CELL_GROUP_EXPERIMENT_ACCESSION_WITH_MARKER_GENES, "10");
```

Comment:
Please use the input parameter `k` instead of a static value for tests.

Reply:
I would replace this hardcode value '10' with a random k value once I update fixtures properly currently, I added very
 few records, without hardcode value tests would fail if I pass random values of K. Currently, I am working on that story to generate fixtures, I need to sort out SCXA local DB dump issues before running script.

Reply:
I just added a comment in the code about this for future reference.